programmers/자릿수-구하기.js

```diff
@@ -0,0 +1,8 @@
+function solution(n) {
+    let answer = 0;
+    let nstr = n.toString();
```

Comment:
변수 타입 const를 1순위로 생각하고 선언해주기, 후에 '=' 연산자를 통해 값이 변한다면 let으로 바꿔주기
그리고 카멜표기법 신경써주기! 
```suggestion
    const nStr = n.toString();
    let answer = 0;
```